lib/ansible/modules/windows/win_service.py

```diff
@@ -229,6 +243,18 @@
     - service1
     - service2
     dependency_action: remove
+
+- name: Set recovery actions
+  win_service:
+    name: service name
+    recovery_reset_interval: 300
+    recovery_actions:
+      - action: restart
+        delay: 60000
```

Comment:
When documenting these sub options make sure you specify the units, it looks like this is in milliseconds but I'm not 100% sure.